src/plugins/discover/public/application/main/services/discover_state.ts

```diff
@@ -235,7 +225,7 @@ export function getDiscoverStateContainer({
   /**
    * Internal State Container, state that's not persisted and not part of the URL
    */
-  const internalStateContainer = getInternalStateContainer();
+  const internalStateContainer = getSharedStateContainer();
```

Comment:
Can we rename to `sharedStateContainer` for consistency?

Reply:
definitely!